src/native/libs/System.Globalization.Native/pal_localeNumberData.c

```diff
@@ -88,12 +88,20 @@ static char* NormalizeNumericPattern(const UChar* srcPattern, int isNegative)
     {
         int length = (iEnd - iStart) + 2;
         destPattern = (char*)calloc((size_t)length, sizeof(char));
+        if (!destPattern)
+        {
+            return NULL;
```

Comment:
Does the called handle null correctly?

Reply:
Add if clause in `GetNumericPattern`. Thank you.